### DISCUSSION TITLE
La data di modifica del documento `updated_at` non viene aggiornata modificando le righe

### DISCUSSION THREAD

**Discussion by d-one-lorenzo** [fattureincloud/api#435] [Q&A]
Ho notato, facendo delle prove in particolare su un rapporto d'intervento `"work_report"`, che il campo `updated_at` ritornato dalle API di lettura dei documenti non viene aggiornato quando vengono fatte delle modifiche sulle righe (ad esempio i nomi o le descrizioni degli articoli) che non modificano altri campi del documento. Questo ovviamente influisce anche sul filtro API `updated_at >=` che uso per elencare i documenti che bisogna ri-sincronizzare nel mio applicativo. Mi sfugge qualcosa, o è un problema che può essere corretto lato FiC? Grazie.

**Comment by valmoz** [comment]
Buongiorno @d-one-lorenzo 
come dice lei, per motivi tecnici il campo updated_at viene aggiornato solo in corrispondenza dei parametri principali della fattura.

Per effettuare la sincronizzazione dei documenti sul suo applicativo le consigliamo di utilizzare i [webhooks](https://developers.fattureincloud.it/docs/webhooks/), che sono stati sviluppati appositamente per notificare le applicazioni esterne della necessità di sincronizzare lo stato di un determinato documento.

A disposizione
Mauro

**Comment by d-one-lorenzo** [comment]
Vedo che i webhooks sono ancora in beta chiusa, infatti abbiamo evitato di utilizzarli sin dall'inizio dello sviluppo. Non c'è un'altra strada percorribile che ci eviti di dover riscrivere l'applicativo?

**Reply by valmoz** [reply]
Al momento no. Valuteremo se possibile trovare una strada alternativa.
Nel caso aggiorneremo questa discussione
